### ISSUE TITLE
`fix_metadata` sometimes only works in serial (at least) for CMIP3

### ISSUE THREAD

**Issue by zklaus** [ESMValGroup/ESMValCore#1247]
**Describe the bug**
The `fix_metadata` preprocessor step can hang if executed in parallel on CMIP3 data. This is illustrated in `recipe_bock20jgr.yml`.

**Comment by bouweandela** [comment]
This looks like a duplicate of #430

**Comment by schlunma** [comment]
Moving this to v2.6 since there is not open PR yet.

**Comment by bouweandela** [comment]
Closing this because it is a duplicate.
